src/frontend/js/widgets/Dashboard/components/DashboardItem/Order/OrderStateMessage/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { captureException } from '@sentry/browser';
+import { FormattedMessage } from 'react-intl';
+
+import { Order, OrderState } from 'types/Joanie';
+import { StringHelper } from 'utils/StringHelper';
+
+export const messages = {
+  statusDraft: {
+    id: 'components.DashboardItemOrder.statusDraft',
```

Comment:
This comment applies on all messages
```suggestion
    id: 'components.OrderStateMessage.statusDraft',
```